evennia/accounts/accounts.py

```diff
@@ -275,11 +275,11 @@ def puppet_object(self, session, obj):
             raise RuntimeError("Session not found")
         if self.get_puppet(session) == obj:
             # already puppeting this object
-            self.msg("You are already puppeting this object.")
+            self.msg(_("You are already puppeting this object."))
             return
         if not obj.access(self, "puppet"):
             # no access
-            self.msg(f"You don't have permission to puppet '{obj.key}'.")
+            self.msg(_("You don't have permission to puppet '{key}'.".format(key=obj.key)))
```

Comment:
Mm, this is not how i18n works. The formatting must be applied _after_ the translation. So it must look like this: 

```python
self.msg(_("You don't have permission to puppet '{key}'.").format(key=obj.key))
```

Note the placement of the format outside the `_()` call.

Reply:
oh! that makes more sense.